.github/workflows/scorecard-analysis.yml

```diff
@@ -18,8 +18,6 @@ jobs:
     
     steps:
       - name: "Checkout code"
-        # Scorecard automatically uses the default branch for non-pull-request events.
```

Comment:
I tried to be smart and I was not! We use the policy file as input but it's not present unless we check out the repo first!